package.json

```diff
@@ -31,9 +31,8 @@
     "express": "^4.18.1",
     "js-yaml": "^4.1.0",
     "nanoid": "^4.0.0",
-    "pug": "^3.0.2",
-    "request": "2.88.2",
-    "request-promise": "4.2.6"
+    "node-fetch": "2.6.9",
```

Comment:
this should be built into node directly I believe you shouldn't need this on node 16

Reply:
ah ok, got you. I have removed now